### DISCUSSION TITLE
Connection lost with server ( Client v1.5.5.1 )

### DISCUSSION THREAD

**Discussion by JerryEnt** [fedorovvl/tso_client#215] [General]
Every first time I started the client, I got a connection error with the server.
After restarting the client, everything works fine.
Is there a way to tackle this problem?

**Comment by fedorovvl** [comment]
same behavior on latest build? (latest build badge on main repo page)

**Reply by JerryEnt** [reply]
I downloaded the latest yesterday 19:3 (GMT +1)

**Reply by fedorovvl** [reply]
latest build != latest release
try with it

**Reply by JerryEnt** [reply]
I download it and will see tomorrow. Because this problem is only the first start.  :)

**Reply by JerryEnt** [reply]
Update now to 1.5.5.2 same problem.

**Reply by JerryEnt** [reply]
Also got complaints from Guild members with the same problem.  Is it possible this is a problem with the Dutch server?

**Reply by fedorovvl** [reply]
can u post a screenshot with error?

**Comment by JerryEnt** [comment]
![ApplicationFrameHost_s3P0VizfEl](https://user-images.githubusercontent.com/32734416/224118625-1093a0dd-9c5b-43ed-8edb-64b9cb25ef0f.gif)

**Reply by fedorovvl** [reply]
looks like u guys have problems with gameservers =)) 
i'll try to check

**Reply by JerryEnt** [reply]
Great!

**Reply by fedorovvl** [reply]
i even cant create new account on NL tso site for checking =))
hangs on uplaylogin

**Reply by JerryEnt** [reply]
We NL is too small for BB :)))

**Comment by tupiacy** [comment]
Hi, maybe you will know something about below error ? showsup right after start
**error parsing shopconfig.xml: typeerror: error #1009**

Thx

**Reply by fedorovvl** [reply]
swf can't parse config file =)
Error #1009: Cannot access a property or method of a null object reference
maybe u have smth that not exist in shopconfig

**Reply by tupiacy** [reply]
I dont get it :)
what may not be in shopconfig?

**Reply by tupiacy** [reply]
Fixed, that was about cache files ;)
